option optimize_for = LITE_RUNTIME;

package TransportBuffers;

message Instruction {
  optional uint64 old_num = 1;
  optional uint64 new_num = 2;
  optional uint64 ack_num = 3;
  optional uint64 throwaway_num = 4;

  optional bytes diff = 5;
}
